packages/contracts/src/managers/ZKEIP1271Manager.sol

```diff
@@ -36,7 +36,7 @@ abstract contract ZKEIP1271Manager is ZKAuth, ZKOwnerManager {
      */
     function _isValidSignature(bytes32 hash, bytes calldata signature)
         internal
-        view
+        pure
```

Comment:
This function is not actually defined in EIP.
Could you pls let me know why did you chose pure not view

Reply:
I made these modifications following the Warning messages that were output when I compiled the code in my local environment.

```
> forge build
[⠘] Compiling...
[⠰] Compiling 2 files with 0.8.20
[⠔] Solc 0.8.20 finished in 28.71s
Compiler run successful with warnings:
Warning (2018): Function state mutability can be restricted to pure
  --> src/managers/ZKEIP1271Manager.sol:37:5:
   |
37 |     function _isValidSignature(bytes32 hash, bytes calldata signature)
   |     ^ (Relevant source part starts here and spans across multiple lines).
```

Regarding the choice of visibility modifiers, I considered pure to be appropriate since the current code does not access any state variables in contract.

If it's anticipated that the todo sections you mentioned in the comments will involve accessing state variables, then it might be acceptable to ignore this warning. How do you think about it?